### PR TITLE
Change cli flag

### PR DESCRIPTION
**Pull request by dongrie**
Change cli flag
`token-address` to `denom`

### CODE REVIEW

File: tests/cases/eth2eth/scripts/test-tx [hunk 1]
```diff
@@ -47,14 +49,14 @@ show_balance() {
   user_wallet=$3
   token_address=$4
```
Comment (siburu) [inline tests/cases/eth2eth/scripts/test-tx:50]: token_address => denom

File: tests/cases/eth2eth/scripts/test-tx [hunk 2]
```diff
@@ -75,7 +77,7 @@ ${RLY} tx acks ${PATH_NAME}
 sleep ${TX_INTERNAL}
 
 # create chain b token address
-CHAIN_B_TOKEN_ADDRESS="${PORT_ID}/${CHANNEL_ID}/${SIMPLE_TOKEN_ADDRESS}"
+CHAIN_B_TOKEN_ADDRESS="${PORT_ID}/${CHANNEL_ID}/${DENOM}"
```
Comment (siburu) [inline tests/cases/eth2eth/scripts/test-tx:80]: CHAIN_B_TOKEN_ADDRESS => CHAIN_B_DENOM

**Comment by siburu** [comment]
You should add a tag to `ethereum-ics20-cli` and specify the tag on git-clone.

File: tests/cases/eth2eth/scripts/test-tx (hunk 1 above)
Comment (dongrie) [reply tests/cases/eth2eth/scripts/test-tx]: Fixed 
https://github.com/datachainlab/yui-relayer-build/pull/11/commits/de386fe2ddf9cbece129a569e30f2a192638c8a5

File: tests/cases/eth2eth/scripts/test-tx (hunk 2 above)
Comment (dongrie) [reply tests/cases/eth2eth/scripts/test-tx]: Fixed
https://github.com/datachainlab/yui-relayer-build/pull/11/commits/de386fe2ddf9cbece129a569e30f2a192638c8a5

**Comment by dongrie** [comment]
@siburu Create Release tag v0.0.2, and use that
https://github.com/datachainlab/ethereum-ics20-cli/releases/tag/v0.0.2
https://github.com/datachainlab/yui-relayer-build/pull/11/commits/7a8b54723c3d78c34bba1d76b1c2ab83b0b7845c

**Review by siburu — APPROVED** [review]

**Pull request merged**